src/app/component/user/components/profile/profile-widget/profile-header/profile-header.component.ts

```diff
@@ -30,6 +30,13 @@ export class ProfileHeaderComponent implements OnInit, OnDestroy {
     this.buildSocialNetworksChart();
   }
 
+  get checkUserCredo() {
```

Comment:
please add type of returned value 
get checkUserCredo(): number {....

Reply:
Done